[2022-07-03 13:47:37,309] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: our_first_dag_v4.second_task manual__2022-07-03T13:47:33.061899+00:00 [queued]>
[2022-07-03 13:47:37,336] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: our_first_dag_v4.second_task manual__2022-07-03T13:47:33.061899+00:00 [queued]>
[2022-07-03 13:47:37,341] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-03 13:47:37,344] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2022-07-03 13:47:37,346] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-03 13:47:37,395] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): second_task> on 2022-07-03 13:47:33.061899+00:00
[2022-07-03 13:47:37,432] {standard_task_runner.py:52} INFO - Started process 1578 to run task
[2022-07-03 13:47:37,474] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v4', 'second_task', 'manual__2022-07-03T13:47:33.061899+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpm39jvxi0', '--error-file', '/tmp/tmp67z8je2y']
[2022-07-03 13:47:37,488] {standard_task_runner.py:80} INFO - Job 10: Subtask second_task
[2022-07-03 13:47:37,874] {task_command.py:370} INFO - Running <TaskInstance: our_first_dag_v4.second_task manual__2022-07-03T13:47:33.061899+00:00 [running]> on host 677d749653ef
[2022-07-03 13:47:38,234] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v4
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-07-03T13:47:33.061899+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-03T13:47:33.061899+00:00
[2022-07-03 13:47:38,241] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-07-03 13:47:38,249] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am the second task running after task1']
[2022-07-03 13:47:38,296] {subprocess.py:85} INFO - Output:
[2022-07-03 13:47:38,314] {subprocess.py:92} INFO - hey, I am the second task running after task1
[2022-07-03 13:47:38,320] {subprocess.py:96} INFO - Command exited with return code 0
[2022-07-03 13:47:38,464] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v4, task_id=second_task, execution_date=20220703T134733, start_date=20220703T134737, end_date=20220703T134738
[2022-07-03 13:47:38,568] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-03 13:47:38,699] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
